packages/postcss-config/lib/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+const { adminColorSchemes } = require( '@wordpress/base-styles' );
+
+module.exports = {
+	plugins: () => [
```

Comment:
It's copied over from `bin/packages/post-css-config.js`. I know nothing about how it works 😅